api/src/pcapi/core/offers/models.py

```diff
@@ -77,13 +77,23 @@ class OfferExtraData(typing.TypedDict, total=False):
     visa: str | None
 
     # allocine
-    cast: str | None
+    allocineId: int | None
+    backlink: str | None
+    cast: list[str] | None
     companies: list[dict] | None
-    countries: str | None
+    countries: list[str] | None
+    credits: list[dict] | None
+    data: dict | None
```

Comment:
je suis pas fan d'une classe qui s'appelle `Data` et qui qui a un champ `data`
```suggestion
    eidr: str | None
    productionYear: str | None
```